src/python_bindings/ar/bind_ar_verification.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+namespace python_bindings {
+void BindArVerification(pybind11::module_& main_module);
```

Comment:
Since we use `AR` in `ARAlgorithm`, let's use everywhere the same naming convention:
```suggestion
void BindARVerification(pybind11::module_& main_module);
```